v03_pipeline/lib/tasks/reference_data/updated_cached_reference_dataset_query.py

```diff
@@ -87,6 +88,7 @@ def create_table(self) -> hl.Table:
         if self.crdq.query_raw_dataset:
             query_ht = import_ht_from_config_path(
                 CONFIG[dataset][self.reference_genome.v02_value],
+                dataset,
```

Comment:
since I mock `import_ht_from_config_path` in tests I didn't catch this until later :/

Reply:
we really need type checking :/